_help/trouble-signing-in/forgot-your-email-address.md

```diff
@@ -1,8 +0,0 @@
----
-order: 5
-redirect_from:
-- /help/signing-in/i-forgot-which-email-address-i-used-to-create-an-account/
-- /help/signing-in/forgot-my-email-address/
```

Comment:
From the build failure, I think we need to keep these from the various pages, or at least move them to another page, maybe `/help`?

Reply:
I redirected them to /help/trouble-signing-in/overview. Should pass now.